### PR TITLE
docs(pie-checkbox): DSW-000 checkbox and checkbox-group from Alpha to Beta

### PR DESCRIPTION
**Pull request by dandel10n**
Feature url: https://pr1729-docs.pie.design/components/checkbox/

Screenshot 2024-08-26 at 14 03 33

Screenshot 2024-08-26 at 14 04 45

Screenshot 2024-08-26 at 14 04 57

### CODE REVIEW

**Review by maledr5 — DISMISSED** [review]

File: .changeset/real-bobcats-run.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+"@justeattakeaway/pie-checkbox-group": patch
+"@justeattakeaway/pie-checkbox": patch
+"pie-docs": patch
+---
+
+checkbox and checkbox-group components from alfa to beta
```
Comment (leksaBoiko) [inline .changeset/real-bobcats-run.md:7]:
```suggestion
[Updated] - checkbox and checkbox-group components from alfa to beta
```

Comment (xander-marjoram) [reply .changeset/real-bobcats-run.md:7]: Also "alpha" not "alfa" 😄

Comment (leksaBoiko) [reply .changeset/real-bobcats-run.md:7]: I'll update suggestion to fix this once

Comment (dandel10n) [reply .changeset/real-bobcats-run.md]: Oops, sorry, one day I'll start to compose sentences in proper english, but it is not guaranteed 😄

**Review by leksaBoiko — APPROVED** [review]

**Review by xander-marjoram — APPROVED** [review]

**Pull request merged**